packages/app-store/googlevideo/components/ExistingGoogleCal.tsx

```diff
@@ -0,0 +1,52 @@
+import { Trans } from "next-i18next";
+import { useState } from "react";
+
+import { CAL_URL } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc";
+import { Icon, SkeletonText } from "@calcom/ui";
+
+const ExistingGoogleCal = () => {
+  const { t } = useLocale();
+  const [firstGoogleCal, setFirstGoogleCal] = useState<string | undefined>();
+
+  const query = trpc.viewer.connectedCalendars.useQuery(undefined, {
+    onSuccess: (data) => {
+      const googleCalQuery = data?.connectedCalendars.find(
+        (calendar) => calendar.primary?.integration === "google_calendar"
+      );
+      if (googleCalQuery?.primary) setFirstGoogleCal(googleCalQuery.primary?.externalId);
+    },
+  });
```

Comment:
Since all tRPC calls are batched this will render the correct message as the same time the install button is enabled

Reply:
Can we add a getConnectedCalendarById so we can fetch the single calendar we need here? I'm all about not transferring more data than needed

Reply:
We're just checking for the presence of an installed GCal. Is it worth adding that as its own endpoint?

Reply:
Does it make sense to fetch all calendars just to check if we have one gCal installed?